src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Tests/LocationOptionMetaRowTests.cs

```diff
@@ -11,27 +11,25 @@ public void ContainsKeyProperties()
         {
             HashSet<string> excludedProperties =
             [
-                nameof(LocationOptionMetaRow.Id),
-                nameof(LocationOptionMetaRow.PublicId),
+                $"{nameof(LocationOptionMetaRow.Id)} value",
             ];
 
             var expectedProperties = typeof(LocationOptionMetaRow)
                 .GetProperties()
-                .Select(p => p.Name)
+                .Select(p => $"{p.Name} value")
                 .Except(excludedProperties)
                 .ToHashSet();
 
             var optionRow = new LocationOptionMetaRow
             {
                 Id = 1,
-                PublicId = nameof(LocationOptionMetaRow.PublicId),
-                Type = nameof(LocationOptionMetaRow.Type),
-                Label = nameof(LocationOptionMetaRow.Label),
-                Code = nameof(LocationOptionMetaRow.Code),
-                OldCode = nameof(LocationOptionMetaRow.OldCode),
-                Urn = nameof(LocationOptionMetaRow.Urn),
-                LaEstab = nameof(LocationOptionMetaRow.LaEstab),
-                Ukprn = nameof(LocationOptionMetaRow.Ukprn),
+                Type = nameof(LocationOptionMetaRow.Type) + " value",
```

Comment:
Are we adding this `" value"` on the end to check for spaces etc?

Reply:
I'm just wanting to make sure it's not just using nameof(field) as the method-under-test's output, so adding something that definitely proves that it's using the values of those fields and not just the names of the fields.